releasenotes/notes/deprecate-qiskit-ibmq-f0dc372526fe0c57.yaml

```diff
@@ -0,0 +1,11 @@
+---
+deprecations:
+  - |
+    The ``qiskit.IBMQ`` object is deprecated. This alias object lazily redirects
+    attribute access to ``qiskit.providers.ibmq.IBMQ``. As the
+    ``qiskit-ibmq-provider`` package has been supersceded by
+    ``qiskit-ibm-provider`` package which maintains its own namespace
+    maintaining this alias is no longer relevant with the new package. If you
+    were relying on the ``qiskit.IBMQ`` alias you should update your usage
+    to use ``qiskit.providers.ibmq.IBMQ`` directly instead (and also consider
+    migrating to ``qiskit-ibm-provider``).
```

Comment:
Does qiskit-ibm-provider has its own documentation site? Can we link to the migration guide (either the [source file](https://github.com/Qiskit/qiskit-ibm-provider/blob/main/docs/tutorials/Migration_Guide_from_qiskit-ibmq-provider.ipynb) or the rendered page on the documentation site) here?

Reply:
Found the link to the migration guide on the doc site: https://qiskit.org/documentation/partners/qiskit_ibm_provider/tutorials/Migration_Guide_from_qiskit-ibmq-provider.html

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/8080/commits/c2d21b435435cc097186e684c78bbecf1004e281